alembic/versions/6496fb7766e3_added_new_apriori_antenna_table.py

```diff
@@ -0,0 +1,69 @@
+"""added new apriori_antenna table
+
+Revision ID: 6496fb7766e3
+Revises: 5fbb4d08569a
+Create Date: 2019-04-12 02:05:04.624693+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '6496fb7766e3'
+down_revision = '5fbb4d08569a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('apriori_antenna',
+    sa.Column('antenna', sa.Text(), nullable=False),
+    sa.Column('start_gpstime', sa.BigInteger(), nullable=False),
+    sa.Column('status', sa.Text(), nullable=False),
+    sa.PrimaryKeyConstraint('antenna', 'start_gpstime')
+    )
+    op.drop_table('paper_temperatures')
```

Comment:
we definitely want to remove these and the create commands below.

Reply:
Why did those show up I wonder?  I was up-to-date.  I have removed those.

Reply:
They always show up now because we've removed the definitions of the tables (the tables are deprecated) but we don't want to delete the info in them. So every alembic version tries to take them out and we have to remove that by hand.

Reply:
I'm actually a little surprised your dubitable table isn't showing up in here -- I'd expect that alembic would be trying to drop it since you've removed the definition for that table...